Am/chore/doc casting (#1261)
zama-ai/tfhe-rs: tfhe/docs/guides/zk-pok.md

lto = "fat"
```
and by building the code for the native CPU architecture and in release mode, e.g. by calling `RUSTFLAGS="-C target-cpu=native" cargo run --release`.

{% hint style="info" %}

> You can choose a more costly proof with `ZkComputeLoad::Proof` which has a faster verification time or the opposite with `ZkComputeLoad::Verify` for a faster proof and a slower verify.

Reviewer: ```suggestion
You can choose a more costly proof with `ZkComputeLoad::Proof`, which has a faster verification time.  Alternatively, you can select `ZkComputeLoad::Verify` for a faster proof and slower verification.
{% endhint %}
```

{% endhint %}

## Using dedicated Compact Public Key parameters

> **TFHE-rs** offers the ability to use dedicated parameters for the Compact Public Key encryption which can allow to reduce the size of encrypted data and speed-up the zero-knowledge proof computation.

Reviewer: ```suggestion
You can use dedicated parameters for the Compact Public Key encryption to reduce the size of encrypted data and speed up the zero-knowledge proof computation.
```

> This works essentially in the same way as before, indicating the dedicated parameters being the only additional step needed:

Reviewer: ```suggestion
This works essentially in the same way as before. Additionally, you need to indicate the dedicated parameters:
```
